lib/capybara/spec/session/node_spec.rb

```diff
@@ -682,6 +682,15 @@
       expect(@session).to have_xpath('//div[contains(., "HTML5 Dropped string: text/plain Some dropped text")]')
     end
 
+    it 'can drop a pathname' do
+      @session.visit('/with_js')
+      target = @session.find('//div[@id="drop_html5"]')
+      target.drop(
+        Pathname.new with_os_path_separators(File.expand_path('../fixtures/capybara.jpg', File.dirname(__FILE__)))
+      )
+      expect(@session).to have_xpath('//div[contains(., "HTML5 Dropped file: capybara.jpg")]')
+    end
```

Comment:
Added this test to prevent a regression, same as the one above but passing a `Pathname` instead.